webapp/src/components/jira_instance_and_project_selector/jira_instance_and_project_selector.test.tsx

```diff
@@ -26,7 +26,9 @@ describe('components/JiraInstanceAndProjectSelector', () => {
         connectedInstances: [{instance_id: 'instance1', type: InstanceType.CLOUD}, {instance_id: 'instance2', type: InstanceType.SERVER}],
         defaultUserInstanceID: '',
         fetchJiraProjectMetadata: jest.fn().mockResolvedValue({data: {
-            default_project_key: 'TEST',
+            default_field_values: {
+                project_key: 'TEST',
```

Comment:
should we include other fields as well?

Reply:
No, only these fields were requested for now. Will update if asked.